### ISSUE TITLE
legend custom list

### ISSUE THREAD

**Issue by dockstreet** [timelyportfolio/sunburstR#73]
Hi,

Can the legend be customized to include a data frame to match colors instead of all of the elements?

**Comment by timelyportfolio** [comment]
Not really, or at least not in an easy way.  The answer is a little more complicated.  If I get some time, I might demonstrate how to achieve with [`d3-legend`](http://d3-legend.susielu.com/).
